lib/handlers/compile.js

```diff
@@ -235,6 +235,11 @@ class CompileHandler {
             _.each((req.query.removeFilters || "").split(","), filter => {
                 if (filter) delete filters[filter];
             });
+            // Ask for asm not to be returned
+            if (req.query.skipAsm === "true") {
+                backendOptions = { skipAsm: true };
```

Comment:
Maybe this should update the existing backendoptions instead of replace them? (it might be surprising later on someday?) Super minor nitpick though!